modules/ding_ddbasic/ding_ddbasic.info

```diff
@@ -0,0 +1,5 @@
+name = DDBasic theme module
+description = Module that help integrate the ddbasic theme.
+package = "Ding!"
+core = 7.x
+dependencies[] = ding_ddbasic_opening_hours
```

Comment:
It's just something that wwe want to enable when ding_ddbasic is enabled.

Should we add it to the ding2.info dependencies instead?